### DISCUSSION TITLE
[Help]0.000000000001 Is this an attack？

### DISCUSSION THREAD

**Discussion by JunHuiJing** [Chia-Network/chia-blockchain#10379] [Support]
### What happened?

Non-stop payment, 0.00000000000 1. I don't know if this will affect the income of farming. Please help me!

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by luis360** [comment]
I think this dust attack，this project is really rubbish, the developer is responsible at all, my transfer transaction has never been able to be on the chain

**Comment by DrakeFS** [comment]
This will not affect your farming unless your full node cannot keep synced.

You can add a fee of at least  .00005 XCH  to make sure a transaction goes through during a dust storm.

**Comment by shaneo257** [comment]
> I think this dust attack，this project is really rubbish, the developer is responsible at all, my transfer transaction has never been able to be on the chain

my transaction went through, after almost 24hrs with no fee, its just delayed, it you want priority you need to add at least 0.00005 for the fee
